### DISCUSSION TITLE
toc shortcode: format html vs raw

### DISCUSSION THREAD

**Discussion by jamesbraza** [thegeeklab/hugo-geekdoc#649] [Q&A] [answered]
From https://geekdocs.de/shortcodes/toc/:

- What is the difference between format being `html` or `raw`?
- Should `html` or `raw` be put in quotes for standardization?

Happy to open a docs PR for this

**Comment by xoxys** [comment, marked as answer]
`html` adds a wrapper around it to support dynamic max level configuration and also adds a horizontal line after it while `raw` just adds the pure ToC provides by Hugo.
